commands/import.go

```diff
@@ -146,28 +161,37 @@ func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks
 		}
 	}
 
+	//set cfg.CurrentChunk
 	if userCurrChunk != -1 { // user gave the current chunk via command line
 		// use the user-provided value
 		cfg.CurrentChunk = userCurrChunk
 	} else { // user didn't specify the current chunk
 		if !dbExists {
-			// if the databse doesn't exist, then assume this is the first chunk
+			// if the database doesn't exist then assume this is the first chunk
+			cfg.CurrentChunk = 0
+		} else if deleteOldData && dbExists && !dbIsRolling {
+			// if the user wants to re-import into a non-rolling database
+			// then assume we want to replace the first chunk
 			cfg.CurrentChunk = 0
+		} else if deleteOldData && dbIsRolling {
```

Comment:
In this last iteration I changed the code so `--chunk` isn't required by `--delete`. If `--chunk` isn't specified, but `--rolling` or `-NC` are, the last chunk is replaced.